components/org.wso2.carbon.identity.authenticator.risk.analytics.endpoint/src/gen/java/org/wso2/carbon/identity/authenticator/risk/analytics/endpoint/ApiException.java

```diff
@@ -0,0 +1,9 @@
+package org.wso2.carbon.identity.authenticator.risk.analytics.endpoint;
+//comment
```

Comment:
Add proper class level comment maybe?

Reply:
It is swagger generated class. Removed the class since it is not used.